addons/account/models/account_move_send.py

```diff
@@ -36,7 +36,7 @@ def _get_default_extra_edis(self, move) -> set:
         return {edi_key for edi_key, edi_vals in extra_edis.items() if edi_vals['is_applicable'](move)}
 
     @api.model
-    def _get_default_invoice_edi_format(self, move) -> str:
+    def _get_default_invoice_edi_format(self, move, **context) -> str:
```

Comment:
Make more sense to me to name it `kwargs` to avoid confusion with the odoo "context"